nemo/lightning/resume.py

```diff
@@ -74,6 +74,10 @@ def setup(self, trainer: Union[pl.Trainer, fl.Fabric], model=None):
         if trainer_ckpt_path:
             trainer.ckpt_path = trainer_ckpt_path
             trainer.checkpoint_callback.last_model_path = trainer_ckpt_path
+            # Restore the tokenizer.
+            tokenizer = load_context(trainer_ckpt_path, "model.tokenizer")
```

Comment:
People could disable io-saving in checkpoints, so this could fail?

Reply:
ok, what would be the best way to check whether model.tokenizer is available in the checkpoint?